sdk/python/kubeflow/training/constants/constants.py

```diff
@@ -19,52 +19,56 @@
 APISERVER_TIMEOUT = 120
 
 # TFJob K8S constants
-TFJOB_GROUP = 'kubeflow.org'
-TFJOB_KIND = 'TFJob'
-TFJOB_PLURAL = 'tfjobs'
-TFJOB_VERSION = os.environ.get('TFJOB_VERSION', 'v1')
+TFJOB_GROUP = "kubeflow.org"
+TFJOB_KIND = "TFJob"
```

Comment:
As @anencore94 says in [this](https://github.com/kubeflow/katib/pull/1951#discussion_r970892056), I also think that it would be good to specify `Final[str]`.